CHANGELOG.md

```diff
@@ -5,7 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [unreleased]
+
+## [3.0.0] - 2024-04-03
```

Comment:
Gotcha. I made changes so that `unreleased` was changed back to `Unreleased`.
I also changed the version of this PR to `2.0.0-rc2`.
If these changes were supposed to go under `2.0.0-rc1`, let me know and I'll make  the change.